scipy/stats/_stats.pyx

```diff
@@ -0,0 +1,31 @@
+#cython: boundscheck=False
+#cython: nonecheck=False
+#cython: wraparound=False
+#cython: cdivision=True
+from libc cimport math
+
+
+ctypedef fused real:
+    short
+    int
+    long
+    float
+    double
```

Comment:
Complexes must be handled in Python-land as they are not ordered from the POV of C.  Is there an easy way to check whether a dtype is a real one (i.e. one of uintX_t, intX_t, floatX_t, longdoubleX_t)?